.travis.yml

```diff
@@ -4,12 +4,18 @@ python:
 before_install:
 - sudo apt-get -qq update
 - sudo apt-get install -y tree
+- mkdir -p "${HOME}"/minio/bin
+- mkdir -p "${HOME}"/minio/data
+- wget -nc -O "${HOME}"/minio/bin/minio https://dl.minio.io/server/minio/release/linux-amd64/minio
+- chmod +x "${HOME}"/minio/bin/minio
 install:
 - pip install -r requirements.txt
 script:
+# Export environment variables for s4cmd
+- export S3_SECRET_KEY=aEmMY45evpLAMMhV3GO0c2dI6PfrCqmKEeUuezVJ
```

Comment:
Minio creates a access key and ID for every new server... Hardcoded keys don't work and pop up an error (https://imgur.com/a/tkxvVIy) and are anyway a bad way of handling this. Perhaps using a config file for minio (currently it creates one in ~/.minio), but we can pick from a diff location. https://docs.minio.io/docs/minio-server-configuration-guide